.github/workflows/checkmake.yaml

```diff
@@ -0,0 +1,30 @@
+name: CheckMake
+on:
+  pull_request:
+    branches:
+      - master
+      - release-*
+
+# cancel the in-progress workflow when PR is refreshed.
+concurrency:
+  group: ${{ github.workflow }}-${{ github.event_name == 'pull_request' && github.head_ref || github.sha }}
+  cancel-in-progress: true
+
+permissions:
+  contents: read
+
+jobs:
+  checkmake:
+    name: CheckMake
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@11bd71901bbe5b1630ceea73d27597364c9af683 # v4.2.2
+      - name: install checkmake
+        run: |
+          sudo apt-get update
+          sudo apt-get install -y wget
+          wget https://github.com/mrtazz/checkmake/releases/download/0.2.2/checkmake-0.2.2.linux.amd64
```

Comment:
does it not have a explicitly plugin  likehttps://github.com/rook/rook/blob/master/.github/workflows/auto-assign.yaml#L18C15-L18C75?

Reply:
In rook we have mostly use actions, @travisn for thoughts

Reply:
One advantage of the actions is that the dependabot will automatically update them, otherwise we have to update them manually, which doesn't have often enough. So I do like the idea of using the action instead.

Reply:
@travisn  wrote:
> One advantage of the actions is that the dependabot will automatically update them, 
otherwise we have to update them manually, which doesn't have often enough. So I do like the idea of using the action instead.

Fair. rewritten to use the action. (but reluctantly ... ;)